### ISSUE TITLE
Information on Size

### ISSUE THREAD

**Issue by sunnykinger** [Qualeams/Android-Face-Recognition-with-Deep-Learning-Test-Framework#37]
Hello,
This is not an issue.
I wanted to ask that how did you manage to have a 17 Mb APK for this?
Please share about this.
Thank you

**Comment by sladomic** [comment]
Hi, do you mean why it is so small or so big? It includes the .so files for some libraries (LIBSVM, TensorFlow, OpenCV, ...) and a quantized TF model.

**Comment by sunnykinger** [comment]
No, I mean how come the APK which is on play store is just 17 Mb size, coz when I forked this project and build an APK on my system, the size was around 55 Mb.

**Comment by MarkyVasconcelos** [comment]
sunny, maybe he have the flavor for architectures on GooglePlay

**Comment by sladomic** [comment]
I guess, that the Google Play store variant only includes the required libraries (e.g. armeabi-v7a) whereas your apk includes all of them (...x86, arm-v8a, ...)

**Comment by sunnykinger** [comment]
yes, I think google play store is doing this magic stuff.
